pages/instructorAssessmentAccess/instructorAssessmentAccess.sql

```diff
@@ -57,6 +79,17 @@ FROM
     LEFT JOIN pt_courses AS pt_c ON (pt_c.id = pt_x.course_id)
 WHERE
     a.id = $assessment_id
-    AND ((aar.role > 'Student') IS NOT TRUE)
 ORDER BY
     aar.number;
+
+-- BLOCK assessment_settings
+SELECT
+    a.type AS type,
+    a.auto_close AS auto_close,
+    a.allow_real_time_grading AS allow_real_time_grading,
+    a.multiple_instance AS multiple_instance
+FROM
+    assessments AS a
+WHERE
+    a.id = $assessment_id
+LIMIT 1;
```

Comment:
We should never have multiple assessments with the same ID.

```suggestion
    a.id = $assessment_id;
```

Reply:
This query is actually leftover from a time when I was a bit naive about the system, it is not needed at all...